engine/src/flutter/shell/platform/embedder/embedder.h

```diff
@@ -1629,6 +1629,8 @@ typedef struct {
   double elevation;
   /// Describes how much space the semantics node takes up along the z-axis.
   double thickness;
+  /// A technical identifier for the node.
+  const char* identifier;
```

Comment:
This file has strict ABI compatibility requirements. As a result, this needs to be added as the last member on this struct.

For more information, see the docs at the top of this file:

https://github.com/flutter/flutter/blob/6b17f99216b12c9d81f19d66cff665f8b30c5056/engine/src/flutter/shell/platform/embedder/embedder.h#L12-L50